examples/vectors.py

```diff
@@ -97,9 +98,9 @@ def get_embedding(cls, input: str) -> List[float]:
             cls._model = SentenceTransformer(MODEL_NAME)
         return cast(List[float], list(cls._model.encode(input)))
 
-    def clean(self):
+    def clean(self) -> None:
         # split the content into sentences
-        passages = nltk.sent_tokenize(self.content)
+        passages = cast(List[str], nltk.sent_tokenize(self.content))
```

Comment:
Offtopic, but I would consider in the future other sentence tokenizers like the Unicode one or spaCy (e.g. the NLTK tokenizer fails on `e.g.` :) )